x/concentrated-liquidity/lp_test.go

```diff
@@ -339,18 +336,6 @@ func (s *KeeperTestSuite) TestWithdrawPosition() {
 				amount1Expected: baseCase.amount1Expected.QuoRaw(2), // 2500 usdc
 			},
 		},
-		"position still unfreezing": {
```

Comment:
Is there a test case left that covers early withdrawal and incentives being forfeited?

Reply:
Added a forfeit incentives test case!

https://github.com/osmosis-labs/osmosis/pull/4724/commits/47c582d32701a373e91e1325779773f15a81ef3a